### PR TITLE
k6: add page

### PR DESCRIPTION
**Pull request by mislavcimpersak**
Adding a page for [k6](https://github.com/loadimpact/k6) open source load testing tool.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by mislavcimpersak** [comment]
trailing whitespace fixed in 92132731

File: pages/common/k6.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# k6
+
+> A modern load testing tool for developers and testers in the DevOps era.
```
Comment (agnivade) [inline pages/common/k6.md]: We can remove the "for developers and testers in the DevOps era.". We try to keep the descriptions as formal and objective as possible.

File: pages/common/k6.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Run load test locally:
+
+`k6 run {{script.js}}`
+
+- Run load test locally with set number of virtual users and set duration:
```
Comment (agnivade) [inline pages/common/k6.md]: "with a given number of virtual users and duration"

File: pages/common/k6.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`k6 run --vus {{10}} --duration {{30s}} {{script.js}}`
+
+- Run load test locally with set environment variable:
```
Comment (agnivade) [inline pages/common/k6.md]: with a given environment variable

File: pages/common/k6.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- Run load test locally using InfluxDB to store results:
+
+`k6 run --out influxdb={{http://localhost:8086/k6db}} {{script.js}}`
+
+- Login into cloud service using secret token:
+
+`k6 login cloud --token {{secret}}`
+
+- Run load test using the cloud executioner:
```
Comment (agnivade) [inline pages/common/k6.md]: "executioner" means something else totally. Can we have a better word ?

File: pages/common/k6.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Run load test locally with set environment variable:
+
+`k6 run -e {{HOSTNAME=example.com}} {{script.js}}`
+
+- Run load test locally using InfluxDB to store results:
+
+`k6 run --out influxdb={{http://localhost:8086/k6db}} {{script.js}}`
+
+- Login into cloud service using secret token:
```
Comment (agnivade) [inline pages/common/k6.md]: "Login into" stutters. "Login to" is better.

[the remark this answers is not in the document]
Comment (mislavcimpersak) [reply pages/common/k6.md]: changed in 2c690a29

File: pages/common/k6.md (hunk 1 above)
Comment (mislavcimpersak) [reply pages/common/k6.md]: changed in 4cbf5349

File: pages/common/k6.md (hunk 2 above)
Comment (mislavcimpersak) [reply pages/common/k6.md]: changed in 4cbf5349

File: pages/common/k6.md (hunk 3 above)
Comment (mislavcimpersak) [reply pages/common/k6.md]: changed in 4cbf5349

File: pages/common/k6.md (hunk 4 above)
Comment (mislavcimpersak) [reply pages/common/k6.md]: yeah. agree. also changed in 4cbf5349

File: pages/common/k6.md (hunk 5 above)
Comment (mislavcimpersak) [reply pages/common/k6.md]: changed in 4cbf5349

**Comment by sbrl** [comment]
Thanks, @mislavcimpersak :smiley_cat: 

Btw, the `=` in the extended description of the merge is a typo :P

**Comment by mislavcimpersak** [comment]
Thank you @sbrl @agnivade @pxgamer for reviewing and merging <3

**Comment by sbrl** [comment]
No problem, @mislavcimpersak :smiley_cat:

**Pull request merged**